_includes/code/wcs.authentication.api.key.with.inference.key.mdx

```diff
@@ -3,7 +3,21 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs groupId="languages">
-<TabItem value="py" label="Python">
+<TabItem value="py4" label="Python (v4)">
+
+```python
+import weaviate
+import os
+
+client = weaviate.connect_to_local(
```

Comment:
Would it be preferable to use `.connect_to_wcs` for consistency & also for contrast vs Weaviate API key, since that has been a source of confusion for the users?